src/ducks/reducer.js

```diff
@@ -276,7 +248,9 @@ export default function reducer(state = initialState, action = {}) {
               name,
               cost,
               description,
-              actions
+              actions,
+              needsTarget,
+              target
```

Comment:
why is target in the ref? this ref is used by all cards with the same ref. if you place something here all of them will have the same target.

Reply:
It's a mistake, it's not needed.